lightning/src/chain/chaininterface.rs

```diff
@@ -65,6 +65,13 @@ pub trait ChainWatchInterface: Sync + Send {
 }
 
 /// An interface to send a transaction to the Bitcoin network.
+///
+/// We may pass non-final transaction (like local HTLC-timeout tx)
+/// in case of channel force-closure due to an offchain anomaly. To warn
+/// being bounce off of every network mempools, implementation MUST
```

Comment:
"To warn bounce off" doesn't translate well to english. I think you mean eg "Note that these will not be accepted by current full nodes until they are able to be confirmed in the next block."